src/index.html

```diff
@@ -16,7 +19,11 @@
       href="./styles/index.scss"
```

Comment:
It seems like you're trying to link a SCSS file directly in your HTML. This won't work because browsers don't understand SCSS syntax. You should compile your SCSS into CSS and then link the CSS file here.